browser/css/jssidebar.css

```diff
@@ -264,8 +261,14 @@ button#button2.ui-pushbutton.jsdialog.sidebar {
 	left: -10px;
 }
 
-#ParaPropertyPanel.sidebar.ui-grid #indentfieldbox {
-	width: 158px;
+#TableEditPanel.sidebar.ui-grid #grid1,
+#ParaPropertyPanel.sidebar.ui-grid #grid1 {
+	grid-template-columns: 1fr 1fr !important;
+	column-gap: 16px;
+}
+
+#numberbullet {
+	width: 143px; /* to much conent break grid columns width */
 }
 
```

Comment:
I'm not sure about this yet. I wonder if changing "#ParaPropertyPanel.sidebar.ui-grid #box1.sidebar.ui-grid-cell" in the jssidebar.js  so justify-content is set to stretch would fix it . What do you think?

Reply:
I played around, but the class I added it the simpliest method to align everything and make it look nicely cause each column use the same width. Therefore I made #6455